curriculum/challenges/english/14-responsive-web-design-22/build-a-personal-portfolio-webpage-project/build-a-personal-portfolio-webpage.md

```diff
@@ -135,6 +135,7 @@ assert.isAtLeast(new __helpers.CSSHelp(document).getCSSRules('media')?.length, 1
 The height of your `#welcome-section` element should equal the height of the viewport.
 
 ```js
+console.log(window.innerHeight)
```

Comment:
This worked for me?

Reply:
This particular one may have given the correct value, but I think the window is referring to one of the other panes or something. When I was checking the `innerWidth` I was getting like 215, so it seemed like it was the instruction pane or one of the code panes - not sure.